namingsystems/NamingSystem-England-ERSUBRN.xml

```diff
@@ -0,0 +1,29 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-ERSUBRN" />
```

Comment:
Why is the resource type-NamingSystem, included as part of the ID?
The convention is England-[BusinessNames]. Also, the business name should be more indicative of its meaning. Thus, this ID should read England-ERSUBRNumber.
This modification will also affect the file name.